### PR TITLE
Working Joes can no longer eat

### PR DESCRIPTION
**Pull request by Ben10083**
# About the pull request

Mechanically prevents eating and being force-fed as a Joe

# Explain why it's good for the game

While not worried about the Joe itself eating the Burger, it physically does not have anything to feed the burger to (The mouth is mostly aesthetic with a speaker). As such, it makes no sense that you can feed food into a Working Joe, and should not be possible.

# Changelog
:cl:
del: Working Joes can no longer be fed.
code: New trait: Cannot eat. Self-explanatory.
/:cl:

### CODE REVIEW

**Comment by Ben10083** [comment]
image

**Review by BeagleGaming1 — APPROVED** [review]
Code seems good

File: code/modules/mob/living/carbon/human/species/synthetic.dm [hunk 1]
```diff
@@ -110,8 +110,8 @@
 	name = SYNTH_WORKING_JOE
 	name_plural = "Working Joes"
 	uses_ethnicity = FALSE
-	burn_mod = 0.65 // made for hazardous environments, withstanding temperatures up to 1210 degrees
-	mob_inherent_traits = list(TRAIT_SUPER_STRONG, TRAIT_INTENT_EYES, TRAIT_EMOTE_CD_EXEMPT)
+  burn_mod = 0.65 // made for hazardous environments, withstanding temperatures up to 1210 degrees
```
Comment (BeagleGaming1) [inline code/modules/mob/living/carbon/human/species/synthetic.dm:113]: You undid some changes here, and made it 2 spaces instead of a tab

**Review by BeagleGaming1 — CHANGES_REQUESTED** [review]

**Review by BeagleGaming1 — APPROVED** [review]

**Review by harryob — CHANGES_REQUESTED** [review]

**Review by harryob — APPROVED** [review]

**Comment by Hopekz** [comment]
please do not feed Seegson property

**Pull request merged**